calcom/cal.com
# feat: add loading state for buttons on settings page

## What does this PR do?

This PR adds loading spinners while performing update mutations across following sections on the settings page:

- Password
- Webhooks
- General
- Appearance
- Two Factor auth
- Impersonation

Fixes [#10539 (issue)](https://github.com/calcom/cal.com/issues/10539)

## Type of change


## How should this be tested?

Go to settings and check the loading state of these buttons:

- Password
- Webhooks
- General
- Appearance
- Two Factor auth
- Impersonation

## Mandatory Tasks


## Checklist



- I have read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code does follow the style guidelines of this project
- I have checked if my PR needs changes to the documentation
- I have checked if my changes generate no new warnings
